src/qt/bitcoin.cpp

```diff
@@ -466,6 +468,10 @@ int GuiMain(int argc, char* argv[])
     QApplication::setOrganizationName(QAPP_ORG_NAME);
     QApplication::setOrganizationDomain(QAPP_ORG_DOMAIN);
     QApplication::setApplicationName(QAPP_APP_NAME_DEFAULT);
+    if (gArgs.GetArg("-guisettings", "") != "") {
```

Comment:
nit: is it really necessary to call `GetArg` twice? We could take `const std::string gui_settings = ...` and use it twice. Pity we can't use C++17 if statement with initializer though...

nit2: it's better to use `! ... .empty()` , often suggested by static analyzers. Less code bloat, no need to actually compare against `const char*`...

Reply:
> nit2: it's better to use ! ... .empty() , often suggested by static analyzers. Less code bloat, no need to actually compare against const char*...

Will add that

> nit: is it really necessary to call GetArg twice? We could take const std::string gui_settings = ... and use it twice. Pity we can't use C++17 if statement with initializer though...

Will add that as well thanks